src/main/drivers/pwm_mapping.c

```diff
@@ -235,6 +235,13 @@ pwmOutputConfiguration_t *pwmInit(drv_pwm_config_t *init)
                 type = MAP_TO_SERVO_OUTPUT;
 #endif
 
+#if defined(RCEXPLORERF3)
```

Comment:
Not as a criticism on this PR, but as a general remark: Has this file been overlooked in the refactoring of target specific code that was done for 3.0, or is there a reason that this code has not been moved into `src/main/target`?